src/pages/2021/sponsorlogistics.mdx

```diff
@@ -2,9 +2,105 @@ import { SiteLayout } from "../../components/layout/SiteLayout";
+
+- **Career and Graduate School Expo -** Think pipe and drape booths, lines of eager attendees, and reviewing resumes / CVs to find your ideal candidates. Don’t be surprised if you see raffle drawings, on-site job/internship offers, iPads collecting contact information, and all the other trappings of your typical career fair. All sponsors with a booth will have the same booth for both Day 1 and Day 2 of the expo.
+
+    - Day 1 - Thursday, October 28, 3:00-7:00 ET (4 hours)
+
+    - Day 2 - Friday, October 29, 5:30-7:30 ET (2 hours)
+
+- **Poster and Hackathon Judging Session - Friday, November 13, 12-3:30 PM PST -** We need folks who are willing to learn more about the amazing research or hackathon project from this year's attendees. No prior experience is required. If you’re interested in being a poster or hackathon judge, please fill out the form linked at the button below.
+
+    - (button: Apply to be a Poster Judge)
+
+    - (button: Apply to be a Hackathon Judge)
+
+**Programming Changes from 2020**
+
+We have expanded our programming session to include 10-minute lightning talks. These fast-paced session will allow our members to talk about a topic that they are passionate about that may range from quick tips for learning a new coding language to understanding the basics of a niche field. This is a great opportunity to learn about what our members are passionate about and how they intersect with LGBT + STEM identities.
+
+# <a name="workshops"></a> Sponsored Workshops
+
+## <a name="submit"></a> 50-minute Sponsored Workshop Submission
+
+50-min Sponsored Workshop is an a la carte sponsor benefit for the conference. This workshop  which will be hosted on webex and listed on Bizzabo. These sessions will be hosted on Thursday, Oct 28, 1:00-3:00 PM ET.  
+
+These workshops can either be pre-recorded or delivered live. 
+
+Please submit what you'd like to present no later than October 1st. Note, if you'd like to use a workshop that was previously submitted in the general Request for Programming, please let us know by emailing sponsorship@ostem.org
+
+- (button: Submit Your Premium Workshop Information Here)
+
+## <a name="submit-ideas"></a> What should you submit?
+
+While you’re welcome to submit an info session type program, we encourage you to present a program that really sparks participants’ interest, has a catchy program description, and shares more about the awesome innovation along STEM or LGBTQIA advocacy and inclusion. Filling in the gaps in our selected programming is a great way to accomplish that!
+
+## <a name="present-ideas"></a> Are you stuck and don't know what to present?
+
+We want you to be successful and we’re here to help! If you’re really stuck and you want some suggestions on your program, email sponsorship@ostem.org and we’ll work through some ideas with you. Just remember to reach out soon so that we can still submit your content by the October 1 deadline.
+
+# Organization Logo Requirements
```

Comment:
There's nothing below this heading. If we don't have the content ready yet, just omit it or comment it out for now.

Reply:
There is content under Org logo. Please see here on the notion page: https://www.notion.so/ostem/2021-Conference-Sponsor-Logistics-Webpage-bff77052c5854738b68ede973af79c57

![image](https://user-images.githubusercontent.com/88102476/129803010-61894c63-fec4-4903-8695-9c2c95a73f78.png)